internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,36 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
-)
 
-var (
-	StepLength = 0.65 // длина шага в метрах
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
+var StepLength = 0.65 // длина шага в метрах
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	pieces := strings.Split(data, ",")
+	if len(pieces) != 2 {
+		return 0, 0, errors.New("Incorrect data received")
+	}
+
+	steps, err1 := strconv.Atoi(pieces[0])
+	if err1 != nil {
+		return 0, 0, errors.New("Error converting string to number")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```